### ISSUE TITLE
Define My Loss Function: Loss1 : y_true - y_pred, loss2: output of a layer1 - output of anther model 

### ISSUE THREAD

**Issue by xinghuokang** [keras-team/keras#8119]
As the title description, Loss1 is defined by (y_true -y_pred ), and loss2 is defined by (dynamic output of a layer - output of a complete model(contain conv layer,fcl layer)), how to achieve the idea by keras?
